blog.md

```diff
@@ -0,0 +1,100 @@
+It puts the primary focus on branches (rather than commits), their relationship to each other,
+and their relationship to the remote repository.
+Moreover, it provides both diagnostic and corrective functionalities
+for the shape of the repository and the mentioned relationships.
+It finds an exceptional application in maintaining linear Git history.
+
+
+### Git Machete Status Graph
+
+![Git Machete Tab with demo repository](gmt_dark.png "Git Machete Tab")
+
+The picture above presents the demo repository _filter-service_ with a few branches.
+It contains two **root branches**: _develop_ and _master_.
+The first of them has two **child branches**: _allow-ownership-link_
+(with another child branch - _build-chain_), and _call-ws_.
+
+Each line contains a name of a branch, an optional custom annotation, and relation to the remote branch.
+
+Without going into details, the **green edge** indicates the desired state -
+the tip of the child branch is a direct descendant of the parent branch tip
+(i.e. all commits reachable from the parent are also reachable from the child).<br/>
+On the other hand, the **red edge** tells us the opposite -
+the tip of the child branch is **not** a direct descendant of the parent branch tip.
+
+In most cases, an edge becomes red after committing some changes to the parent branch (review fixes)
+or pulling new commits to the parent from a remote.
+
+
+### Machete File
+
+![Machete file](machete-file_dark.png "Machete file")
+
+The branches presented within the graph are defined in a very simple configuration file (`.git/machete`),
+which can be easily opened from the plugin's toolbar.
+It supports both syntax highlighting and completion of local branch names.
+The graph is automatically refreshed when `.git/machete` is modified.
+
+Bear in mind that there is no need to edit the machete file manually to set up your branch layout tree.
+Feel free to use `Discover Branch Layout` functionality that will find it out for you automatically.
+It can be performed from `Help -> Find Action...` (default shortcut: `Ctrl + Shift + A`).
+
+
+### Lights, Camera... Actions
+
+The plugin provides a bunch of actions that can help you to manage your repository.
+
+Among the others, there is `Rebase onto parent`
+(that performs interactive rebase of a given branch onto its parent), `Push` and `Pull`.
+Let's have a look at the following example...
+
+![Git Machete - Actions](actions-before_dark.png "Git Machete - Actions - Before")
+
+
+Branch `allow-ownership-link` is **out of sync** (red edge) to its parent - `develop`.
+Furthermore, it is **ahead of** its remote branch (`origin/develop`).
+With Git Machete we can quickly and conveniently lead it to a desired state.
+
+
+First, `Checkout and Rebase Branch Onto Parent`.
+This will perform an interactive rebase from IntelliJ.
+Once the rebase is finished the branch is now **in sync** (green edge) with its parent.
```

Comment:
```suggestion
Once the rebase is finished, the branch is now **in sync** (green edge) with its parent.
```

Reply:
Or even:
```suggestion
Once the rebase is finished, the branch is **in sync** (green edge) with its parent.
```

Having both `Once ...` + `now` in a single sentence sounds weird (same in Polish, try mixing `Dopiero kiedy...` and `teraz`)

Reply:
You're right